src/System.IO.Compression/tests/System.IO.Compression.Tests.csproj

```diff
@@ -45,8 +49,11 @@
       <Link>Common\System\IO\Compression\ZipTestHelper.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
```

Comment:
Right, but this has a `netstandard` configuration which indicates it works *on top* of netstandard. It can expose any API it wants. I don't think you need any new configuraiotn.